semih.js

```diff
@@ -0,0 +1,119 @@
+/*Assignment #1*/
+/*printing hello world in a couple of languages with console.log function */
+console.log("Hello World // English");
+console.log("Merhaba Dünya // Turkish");
+console.log("Привет, мир // Russian");
+console.log("Hallo Wereld // Dutch");
+
+
+/*Assignment #2*/
+/*added \ for escape character*/ 
+console.log('I\'m awesome'); 
+
+
+/*Assignment #3*/
+let x;
+console.log("The value of my variable x will be: 'undefined'");
+console.log(x);
+x = 0;
+console.log("The value of my variable x will be: 0");
+console.log(x);
+
+/*Assignment #4*/
+let y = "some_string";
+console.log("The value of my variable y will be: 'some_string'");
+console.log(y);
+y = "new_string"
+console.log("The value of my variable y will be: 'new_string'");
+console.log(y);
+
+
+/*Assignment #5*/
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let highestvalue;
+
+if (a<z) {
+    highestvalue=z;
+} else {
+    highestvalue=a;
+}
+
+console.log(highestvalue);
+
+
+/*Assignment #6*/
+let items=[];
+console.log("The value of my array will be:'[]'");
+console.log(items);
+let favorite_animals=["horse", "lion", "tiger", "bird", "turtle"];
+console.log(favorite_animals);
+favorite_animals.push("baby pig");
+console.log(favorite_animals);
+
+
+/*Assignment #7*/
+let myString="this is a test";
+console.log(myString);
+let string_length=myString.length;
+console.log(string_length);
```

Comment:
Two things here please:
1. Variable name should follow camelCase as I have suggested above. I see you use the convention while naming `myString`. Wondering why you used underscore for the length variable.
2. You can use `const` keyword here instead of `let` to create these variables.

Reply:
Alright, I'll be extra careful about `const` and camelCase!